tools/cldr-code/src/main/java/org/unicode/cldr/util/CodePointEscaper.java

```diff
@@ -110,6 +110,18 @@ public enum CodePointEscaper {
     public static final UnicodeSet EMOJI_INVISIBLES =
             new UnicodeSet("[\\uFE0F\\U000E0020-\\U000E007F]").freeze();
 
+    /** Includes unassigned, surrogate, noncharacters. Some Cf are visibles! */
+    public static final UnicodeSet INVISIBLES =
```

Comment:
Why are you defining this variant of FORCE_ESCAPE? It removes the following characters:

https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=[%5b%5b%3aDI%3a%5d%5b%3aPat%5fWS%3a%5d%5b%3aWSpace%3a%5d%5b%3aC%3a%5d%5b%3aZ%3a%5d%5cu200B%5cu2060%5d-%5b%5b%3aCc%3a%5d%5b%3aCn%3a%5d%5b%3aZ%3a%5d%5b%3aCs%3a%5d%5b%3aCo%3a%5d%5b%5b%3aCf%3a%5d%2d%5b%5c%5cu0600%2d%5c%5cu0605%5c%5cu06dd%5c%5cu070f%5c%5cu0890%2d%5c%5cu0891%5c%5cu08e2%5c%5cU000110BD%5c%5cU000110CD%5d%5d%5d]

Reply:
I'm surprised that there are `Lo` that are unprintable.

I didn't know FORCE_ESCAPE was the one to use, but I can use that instead. Looks like it uses `[:DI:]`